site/react/useSignMessage.md

```diff
@@ -0,0 +1,41 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: useSignMessage
+  - - meta
+    - name: description
+      content: An overview of the useSignMessage hook
+  - - meta
+    - property: og:description
+      content: An overview of the useSignMessage hook
+  - - meta
+    - name: twitter:title
+      content: useSignMessage
+  - - meta
+    - name: twitter:description
+      content: An overview of the useSignMessage hook
+---
+
+# useSignMessage
+
+The `useSignMessage` hook enables signing a message on behalf of the user's Embedded Account.
```

Comment:
we should call out here that this will use 6492 if the account is not deployed yet